specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/wsn/server.may.send.extended.ping.control.frames/response.rpt

```diff
@@ -0,0 +1,40 @@
+accept tcp://127.0.0.1:8001
+accepted
+connected
+
+# Connection request
+read "GET /echo?.kl=Y HTTP/1.1\r\n"
+read "Upgrade: websocket\r\n"
+read "Connection: Upgrade\r\n"
+read "Host: localhost:8001\r\n"
+read "Origin: http://localhost:8001\r\n"
+read "Sec-WebSocket-Key: nDaimG37f4nUqogPypithw==\r\n"
+read "Sec-WebSocket-Version: 13\r\n"
+read "Sec-WebSocket-Extensions: x-kaazing-ping-pong\r\n"
+read "\r\n"
+
+# 101 Response
+write "HTTP/1.1 101 Web Socket Protocol Handshake\r\n"
+write "Connection: Upgrade\r\n"
+write "Date: Fri, 31 Dec 1999 23:59:59 GMT\r\n"
+write "Sec-WebSocket-Accept: TXCKJSpylVJ7v9ZGLGbt58vypzE=\r\n"
+# The 4 control bytes for ping-pong extension are 0x01010102:
+write "Sec-WebSocket-Extensions: x-kaazing-ping-pong; 01010102"
+write "\r\n"
+write "Server: Kaazing Gateway\r\n"
+write "Upgrade: websocket\r\n"
+write "\r\n"
+
+# binary-encoded balancer directive - NO balance needed
+write [0x82 0x04 0xEF 0x83 0xBF 0x4E]
+
+# Send extended PING
+write [0x81 0x06 0x01 0x01 0x01 0x02 0x09 0x00]
+
+# Expect masked extended PONG response
+#read [0x81 0x86 0x01 0x01 0x01 0x01 0x00 0x00 0x00 0x03 0x0b 0x01]
```

Comment:
This should not be commented out. The client is sending the PONG so server should read it.